### PR TITLE
feat(slides/slide-objects)[OK-116] Add response object methods

### PR DESCRIPTION
**Pull request by marcepanowyy**

### CODE REVIEW

File: backend/src/modules/slide-objects/slide-objects.service.ts [hunk 1]
```diff
@@ -8,82 +8,135 @@ import {
...
   constructor(
     @InjectModel(SlideObject.name)
     private readonly slideObjectModel: Model<SlideObject>,
     private readonly userService: UsersService,
     private readonly slideService: SlidesService,
   ) {}
 
-  async findAll(): Promise<SlideObjectDocument[]> {
-    return this.slideObjectModel.find().exec();
+  async getSlideObjects(
```
Comment (Wajktor13) [inline backend/src/modules/slide-objects/slide-objects.service.ts:23]: What is the use case of this method?

File: backend/src/modules/slide-objects/slide-objects.service.ts [hunk 2]
```diff
@@ -8,82 +8,135 @@ import {
...
-    // const user = await this.userService.findOneById(userId);
+  ): Promise<SlideObjectResponseObject> {
+    return this.deleteSlideObjectById(slideObjectId).then(deletedSlideObject =>
+      this.toResponseSlideObject(deletedSlideObject),
+    );
+  }
+
+  ////////////////////////////////////////////////////////////////////////////////////
 
+  async findSlideObjects(
```
Comment (Wajktor13) [inline backend/src/modules/slide-objects/slide-objects.service.ts:93]: Again, what is the use case?

**Review by Wajktor13 — APPROVED** [review]

File: backend/src/modules/slide-objects/slide-objects.service.ts (hunk 1 above)
Comment (marcepanowyy) [reply backend/src/modules/slide-objects/slide-objects.service.ts:23]: To remain consistency - as for the rest services there are methods called this way (its started out when method findUserById was used by auth service and users controller - i decided to divide the method into two - first one returning the whole document and the second one returning only response object. For the future reference, we might need it as well when operating on slide.service

Comment (Wajktor13) [reply backend/src/modules/slide-objects/slide-objects.service.ts:23]: I don't see the use case for fetching objects from all slides of all boards with pagiantion. I don't think we want to have code that won't be used. However I can live with that so I approve.

**Pull request merged**